client/src/components/DescriptiveTextButton/DescriptiveTextButton.css

```diff
@@ -0,0 +1,15 @@
+.no-account-text {
```

Comment:
Change to `DescriptiveTextButton.modules.css` and make any necessary changes that use this CSS file
Refer to the comment made on the first CSS file that I commented on

Reply:
Fixed (Nestor): changed to .module